src/tribler-core/tribler_core/modules/metadata_store/tests/test_torrent_metadata.py

```diff
@@ -283,7 +283,10 @@ def test_update_properties(self):
         Test the updating of several properties of a TorrentMetadata object
         """
         metadata = self.mds.TorrentMetadata(title='foo', infohash=random_infohash())
+        orig_timestamp = metadata.timestamp
 
         # Test updating the status only
         self.assertEqual(metadata.update_properties({"status": 456}).status, 456)
+        self.assertEqual(orig_timestamp, metadata.timestamp)
         self.assertEqual(metadata.update_properties({"title": "bar"}).title, "bar")
+        self.assertLess(orig_timestamp, metadata.timestamp)
```

Comment:
Just as a minor comment, it would be helpful to make the `metadata.update_properties` call more explicit (e.g., by moving this on its own line). Now it was slightly misleading for me that line 290 and 292 are operating on a different `metadata` object 👍

Reply:
Yep, I agree. I was just a bit too lazy for that :unamused: 

I hope it will become easier to write expressive tests when you finish with you `pytest` refactoring.